### PR TITLE
Implemented Product Details WireFrame

### PR DESCRIPTION
**Pull request by ThinkCreatively**
Implemented wireframe for Product Detail component
Added a material Ui file in porduct-display folder

### CODE REVIEW

File: client/src/components/product-display/MaterialUi.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
```
Comment (Pix3lAssassin) [inline client/src/components/product-display/MaterialUi.jsx:3]: Like the additional file approach

File: client/src/components/product-display/ProductDisplay.jsx [hunk 2]
```diff
@@ -1,14 +1,82 @@
...
+            </Grid>
+          </Grid>
+        </Grid>
+        <Grid className={classes.grid6} item xs={12} container>
+          <Grid className={classes.grid5} item xs={12} container>
+            Description
+          </Grid>
+        </Grid>
+      </Grid>
+    </div>
```
Comment (Pix3lAssassin) [inline client/src/components/product-display/ProductDisplay.jsx:79]: Looks good could use a few less grids lol

**Review by Pix3lAssassin — APPROVED** [review]
Looks good

File: client/src/components/product-display/ProductDisplay.jsx (hunk 2 above)
Comment (mhhutton) [reply client/src/components/product-display/ProductDisplay.jsx:79]: Good start using all Material UI inside parent div

File: client/src/components/product-display/MaterialUi.jsx (hunk 1 above)
Comment (mhhutton) [reply client/src/components/product-display/MaterialUi.jsx:3]: Good start using themes.

**Pull request merged**